src/components/Header.css

```diff
@@ -0,0 +1,36 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+
+.Header {
+  display: flex;
+
+  /* justify-content: left; */
+  background-color: #fff;
+  height: 70px;
+  line-height: 70px;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+.Header_title {
+  font-family: Montserrat;
+  font-size: 20px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  margin-right: 20px;
+}
+
```

Comment:
- [ ] It would be so much better if you take out the default property for the links in the header. 

![tempsnip](https://user-images.githubusercontent.com/64173189/135726731-3dcdcd81-9b86-4b2d-b7ed-8a463efdedb7.png)

Just give them a property of  

```css
  text-decoration: none;
  color: the color;
```

💪